src/nft/components/bag/BagFooter.tsx

```diff
@@ -531,7 +574,12 @@ export const BagFooter = ({ totalEthPrice, bagStatus, fetchAssets, eventProperti
         >
           <Warning>{warningText}</Warning>
```

Comment:
fast follow: update/refresh error state when an item is added/removed
![Screen Shot 2023-02-15 at 13 29 45 ](https://user-images.githubusercontent.com/11512321/219172672-93e9800b-9a18-45c7-ad01-01370493ffb5.png)